htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -596,3 +596,4 @@ create table llx_onlinesignature
   pathoffile				varchar(255)
 )ENGINE=innodb;
 
+ALTER TABLE llx_knowledgemanagement_knowledgerecord ADD default_lang varchar(6)
```

Comment:
Add the ; at end so a new line added by another PR or a merge will not generate syntax error;